automation/run-tests.sh

```diff
@@ -13,10 +13,22 @@ TEST_TYPE_LINT="lint"
 TEST_TYPE_UNIT_PY36="unit_py36"
 TEST_TYPE_UNIT_PY38="unit_py38"
 TEST_TYPE_INTEG="integ"
+TEST_TYPE_INTEG_SLOW="integ_slow"
 
 FEDORA_IMAGE_DEV="nmstate/fedora-nmstate-dev"
 CENTOS_IMAGE_DEV="nmstate/centos8-nmstate-dev"
 
+PYTEST_OPTIONS="
+    --verbose --verbose
```

Comment:
This does not work as expected.

https://travis-ci.com/nmstate/nmstate/jobs/286905321#L895

Looks like we need a ending `\`

Reply:
Fixed! Thanks!